edi_sale_order_import_ubl/__manifest__.py

```diff
@@ -11,7 +11,7 @@
     "website": "https://github.com/OCA/edi",
     "author": "Camptocamp,Odoo Community Association (OCA)",
     "maintainers": ["simahawk"],
-    "depends": ["edi_ubl_oca", "edi_sale_order_import", "sale_order_import_ubl"],
-    "auto_install": True,
+    "depends": ["edi_ubl_oca", "edi_sale_oca", "sale_order_import_ubl"],
+    "auto_install": False,
```

Comment:
Hi a @thienvh332 , why you don't update it like [this one](https://github.com/thienvh332/edi/pull/28/commits/842dd51f4a1fd7fd71b692a00f58af53063ff762)

this module will be replaced by `edi_sale_ubl_oca`